docs/source/en/core/deployment.md

```diff
@@ -50,7 +50,7 @@ Then add `npm scripts` to `package.json`:
 
 Then we are able to use `npm start` and `npm stop` to manage application.
 
-> Note: `egg-scripts` don't support Windows.
+> Note: `egg-scripts` has limited support for Windows, see [#22](https://github.com/eggjs/egg-scripts/pull/22).
```

Comment:
how about wait for fully support, then updated this together ?

Reply:
Sure.

About `limited support`, I just want to put the notice: some people may failed to run `wmic Path win32_process Where "Name = 'node.exe'" Get CommandLine,ProcessId` on windows.